docs/WALKTHROUGH.md

```diff
@@ -0,0 +1,88 @@
+
+When you are ready to test your changes with this ATH, you will need to run `mvn install` on your local copy of Jenkins. This achieves multiple things:
+1) `mvn install` will let you use your local Jenkins version as a dependency, as it generates you a `SNAPSHOT` version. 
+
+   - In the example output below, you can see that I have generated a `2.415-SNAPSHOT` version in my .m2 folder. Now I can use `2.415-SNAPSHOT` as a dependency in another project.
+    ```shell
+    [INFO] --- install:3.1.1:install (default-install) @ jenkins-coverage ---
+    [INFO] Installing C:\Users\julie\jenkins\coverage\target\jenkins-coverage-2.415-SNAPSHOT.pom to C:\Users\julie\.m2\repository\org\jenkins-ci\main\jenkins-coverage\2.415-SNAPSHOT\jenkins-coverage-2.415-SNAPSHOT.pom
+    [INFO] ------------------------------------------------------------------------
+    [INFO] Reactor Summary for Jenkins main module 2.415-SNAPSHOT:
+    [INFO]
+    [INFO] Jenkins main module ................................ SUCCESS [  3.112 s]
+    [INFO] Jenkins BOM ........................................ SUCCESS [  0.066 s]
+    [INFO] Internal SPI for WebSocket ......................... SUCCESS [  4.168 s]
+    [INFO] Jetty 10 implementation for WebSocket .............. SUCCESS [  3.645 s]
+    [INFO] Jenkins cli ........................................ SUCCESS [  7.500 s]
+    [INFO] Jenkins core ....................................... SUCCESS [01:10 min]
+    [INFO] Jenkins war ........................................ SUCCESS [ 28.499 s]
+    [INFO] Tests for Jenkins core ............................. SUCCESS [ 13.749 s]
+    [INFO] Jenkins coverage ................................... SUCCESS [  0.363 s]
+    [INFO] ------------------------------------------------------------------------
+    [INFO] BUILD SUCCESS
+    [INFO] ------------------------------------------------------------------------
+    ```
+
+2) Navigate in your local Jenkins repo to `…\war\target` and you will see a war file for you to use later
+
+### Testing with this ATH
+1) Clone a copy of this ATH repo to your local machine and `cd` into the directory
+2) Open the `pom.xml` in your local copy of the ATH
+3) Change the `<jenkins.version>` to your SNAPSHOT version of jenkins (mine is `2.415-SNAPSHOT`)
+
+> **Note:** There are many ways to run these tests, I will be running the tests using Windows, your commands may differ slightly if using a different OS. See the [README.md](README.md) for alternatives and configuration options.  
+
+4) In a terminal window type `set JENKINS_WAR=C:\Users\julie\jenkins\war\target\jenkins.war` This sets a new environment variable called `JENKINS_WAR` to the path of your copy of the jenkins war *(You created this in the `mvn install step`, change it to your directory path instead of mine)*
```

Comment:
Ok so another way to set, so you definitely do not need to set the Jenkins
Version earlier!